lib/pbench/test/unit/server/test_requests.py

```diff
@@ -160,6 +160,26 @@ def test_bad_extension_upload(client, bad_extension, caplog, server_config):
             for record in caplog.records:
                 assert record.levelname not in ("WARNING", "ERROR", "CRITICAL")
 
+    @staticmethod
+    def test_invalid_authorization_upload(client, caplog, server_config):
+        with client:
+            auth_token = get_pbench_token(client, server_config)
+            # Log the token out
+            response = client.post(
+                f"{server_config.rest_uri}/logout",
+                headers=dict(Authorization="Bearer " + auth_token),
+            )
```

Comment:
Ha; I'd envisioned just making up a random "almost certainly bad" token, not actually logging in and then logging out again to get a guaranteed "good format but invalid" token.

I think _both_ cases are likely worth testing. Though maybe the other is more appropriate for your earlier basic authorization tests and may already be covered.

Reply:
We've got

- filename
  - missing
  - bad extension
- Auth token
  - missing
  - invalidated
- MD5 Hash
  - missing

It seems like there are at least two more cases that we could cover:
- a malformed Auth token
- a mismatched MD5 hash

Reply:
Tracked in #2246.